### ISSUE TITLE
Failing to clean dummy-chunk on some occasions

### ISSUE THREAD

**Issue by Maiquu** [cascornelissen/svg-spritemap-webpack-plugin#81]
Using `webpack 4.x`, `svg-spritemap-webpack-plugin 3.3.5`, Webpack configuration is something like this:
  ```
module.exports = {
      mode: "development",
      output: {
        path: path.resolve(context,"./dist"),
        filename: "code.bundle.js",
        //chunkFilename: "[id].js",
        publicPath: ""
      },
      plugins: [
        new SVGSpritemapPlugin('./editor/assets/svg/*.svg',{
          output: {
            filename: "symbol-defs.svg",
            svg4everybody: false
          },
          sprite: {
            prefix: "icon-"
          }
        })
      ]
  }
```
If I define `output.filename` as a string without any placeholders, dummy module's chunk uses that string as a postfix to define its filename. This causes the internal check to remove the dummy chunk fail. 
```
compilation.chunks.forEach((chunk) => {
  if ( chunk.name !== outputOptions.chunk.name ) {
    return;
  }
  // chunk.name is 'spritemap' while chunk filename is 'spritemap.code.bundle.js' 
  chunk.files.filter((file) => {
    return [`${chunk.name}.js`, `${chunk.name}.js.map`].includes(file);
  }).forEach((file) => {
    delete compilation.assets[file];
  });
});

```
It can be easily fixed if you define a `output.filename` with placeholders or define `output.chunkFilename`. Latter is the one I preffered. Not sure if this is considered nitpicking.

**Comment by cascornelissen** [comment]
@Maiquu, I'm not 100% sure what you're trying to say and I think this is caused by the fact that both the main webpack object and the plugin options take an `output.filename`... Could you clear that up a bit?

**Comment by Maiquu** [comment]
The `output.filename` I mentioned above is from the main webpack object.

**Comment by cascornelissen** [comment]
Ah, I get it now and it should be fixed in `master`, in 8f4090e96 specifically. Could you try installing it directly from there and reporting if that fixes it for you?

```shell
npm install --save-dev cascornelissen/svg-spritemap-webpack-plugin
```

**Comment by Maiquu** [comment]
Yep, no issues spotted. Thanks for your quick response.

**Comment by cascornelissen** [comment]
Thanks for your quick response as well! I'll leave the issue open until I release a new patch version containing the fix ✌️

**Comment by cascornelissen** [comment]
@Maiquu, a fix for this issue has been released in version [`3.3.6`](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/releases/tag/v3.3.6) 🚀
